README.md

```diff
@@ -69,6 +69,27 @@ optional arguments:
                         the request. This is the default behaviour.
 ```
 
+Installation
+=====
+
+Currently, the only way to install / use desec-dns is from source.
+
+You can simply run `python desec.py` as-is, as long as `requests` is installed for your
+python interpreter. Note that some features requre additional dependencies (see above).
```

Comment:
How about this? Deduplicate some information.
```suggestion
You can simply run `python desec.py` as-is, as long as the dependencies (see
[above](#requirements)) are installed for your python interpreter.
```

Reply:
Done